src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -2012,11 +2012,49 @@ public static void DCS_TypeWithXmlElementProperty()
     [Fact]
     public static void DCS_DifferentCollectionsOfSameTypeAsKnownTypes()
     {
-        Assert.Throws<InvalidOperationException>(() => { 
+        Assert.Throws<InvalidOperationException>(() => {
             (new DataContractSerializer(typeof(TypeWithKnownTypesOfCollectionsWithConflictingXmlName))).WriteObject(new MemoryStream(), new TypeWithKnownTypesOfCollectionsWithConflictingXmlName());
         });
     }
 
+    [Fact]
+    public static void DCS_MyPersonSurrogate_Stress()
```

Comment:
The reason that I used multi-iterations here is because the issue does not always repro. It was a matter of odds. The more iterations we run here the more likely we repro the issue. From my experiments, it's half-chance to repro the issue running 5 iterations.

Running 50 iterations does increase the total time significantly. I will reduce this to 5 iterations. In the future, we probably can change the value when we can have dedicated stress tests.